doc/languages-frameworks/python.section.md

```diff
@@ -299,12 +299,12 @@ python3Packages.buildPythonApplication rec {
     hash  = "sha256-Pe229rT0aHwA98s+nTHQMEFKZPo/yw6sot8MivFDvAw=";
   };
 
-  nativeBuildInputs = [
+  build-system = [
```

Comment:
The name of this hints that it would be a string despite it being a list which can contain more than one thing. Also it is not camelCased.